dev-games/ogre/ogre-1.10.12.ebuild

```diff
@@ -13,41 +15,46 @@ LICENSE="MIT public-domain"
 SLOT="0/1.10.0"
 KEYWORDS="~amd64 ~arm ~x86"
 
-IUSE="cg doc double-precision examples +freeimage gl3plus gles2 gles3 ois +opengl profile tools"
+IUSE="beta-components +cache cg doc double-precision egl examples +freeimage
```

Comment:
`beta-components` is used nowhere else in tree; it seems like something that could use the well-known `experimental` use flag?

Reply:
Good point. I used "beta-components" as its a term from the build system configuration.
"experimental", however, is known and descriptive. I'll change that.